go.mod

```diff
@@ -14,11 +14,12 @@ require (
 	github.com/gobuffalo/packr/v2 v2.7.1
 	github.com/golang/groupcache v0.0.0-20191002201903-404acd9df4cc // indirect
 	github.com/golang/mock v1.3.1
-	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/google/go-github v17.0.0+incompatible
```

Comment:
We should use only one version of `go-github` unless there's reason that we have to.